packages/dataviews/src/view-actions.tsx

```diff
@@ -315,7 +315,7 @@ function _ViewActions< Item extends AnyItem >( {
 				<Button
 					size="compact"
 					icon={ settings }
-					label={ __( 'View options' ) }
+					label={ __( 'Screen options' ) }
```

Comment:
Rather than changing the string, I'd recommend providing context for translators that makes it clear that "View" here is a noun:

```suggestion
					label={ _x( 'View options', 'View is used as a noun' ) }
```

Reply:
@tyxla Thanks for the review. Ya it makes more sense to give context rather than changing the whole string. Thanks.